modules/management-console-agent/src/main/java/org/apache/ignite/agent/Agent.java

```diff
@@ -274,6 +274,12 @@ private void connect() {
             return;
         }
 
+        if (F.isEmpty(cfg.getConsoleUris())) {
+            log.info("Skip start Management Console agent on coordinator, because the server URI is not set");
```

Comment:
```suggestion
            log.info("Skip start Management Console agent on coordinator, because the server URI is not set");
            log.info("Check that 'ignite.properties' file contains value for 'ignite.management.console.uris' property");
```

Reply:
How user will change ignite.properties file in ignite-core.jar? I don't think we should add this.

Reply:
When user will read message "Skip start Management Console agent on coordinator, because the server URI is not set" he will have no idea where it was not set.
We need to print message that will help user to understand what to do.

Reply:
Done